tests/bearlib/languages/LanguageMembersTypeTests.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+from coalib.bearlib.languages.Language import Language
+from tests.TestUtilities import LANGUAGE_NAMES
+
+
+class LanguageDefinitionTest(unittest.TestCase):
+
+    def test_comment_delimiters_type(self):
+        for language_name in LANGUAGE_NAMES:
+            version = Language[language_name].get_default_version()
+            language = Language[version]
```

Comment:
Can you put this into `setUp` function, so that these three lines can be reused?
Then in each test case, we can simply do
```python
for language in languages:
    attribute = language.attributes
```